src/com/owncloud/android/authentication/AuthenticatorAsyncTask.java

```diff
@@ -72,18 +72,32 @@ protected RemoteOperationResult doInBackground(Object... params) {
             );
             result = operation.execute(client);
 
+            String permanentLocation = null;
```

Comment:
Maybe a comment here could help to understand the use of this variable. What do you mean with permanent location?